emf/src/main/java/io/lionweb/lioncore/java/emf/EMFModelExporter.java

```diff
@@ -64,7 +64,17 @@ public EObject exportTree(Node root, ReferencesPostponer referencesPostponer) {
                             .collect(Collectors.toList());
                     eObject.eSet(eReference, childrenInEmf);
                   } else {
-                    throw new UnsupportedOperationException();
+                    List<? extends Node> childrenInLW =
+                        root.getChildrenByContainmentName(eReference.getName());
+                    if (childrenInLW.size() > 1) {
+                      throw new IllegalStateException(
+                          "More than one child found, where up to one ");
```

Comment:
I think this error message is hard to understand without the context.

Reply:
Right, I tried updating it